app/javascript/components/pages/MedicationShow.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+
+export default class MedicationShow extends Component {
+  render() {
+    const { medication_name, dose, frequency, time, route, tx, prescribed_by } =
+      this.props.medication;
+    return (
+      <>
+        {/* <h3>Medication Show</h3>
```

Comment:
you shouldn't commit comments. If you need it later make a commit so you can go back to get it.

Reply:
got it, will remove